### ISSUE TITLE
Add support for ROR ID as a type of institutional ID

### ISSUE THREAD

**Issue by joncison** [bio-tools/biotoolsSchema#198]
i.e. alongside GRID id

**Comment by joncison** [comment]
@hansioan  this will go in as "rorid" (alongside "orcidid" and "gridid")

**Comment by hansioan** [comment]
@joncison thanks , will also need a regex for this

**Comment by joncison** [comment]
Yup.  Syntax is a leading 0 followed by 6 characters and a 2-digit checksum, see https://dev.ror.org/facts/

Will paste a XSD style regex here shortly ...

**Comment by joncison** [comment]
``<xs:pattern value="0[0-9a-zA-Z]{6}[0-9]{2}"/>``

**Comment by joncison** [comment]
@hansioan could you pls. paste a JSON style regex here?  thanks.

**Comment by hansioan** [comment]
@joncison I don't know about a JSON style regex, but in the backend I will add the regex:
`^0[0-9a-zA-Z]{6}[0-9]{2}$`
I've basically included the start character `^` and the end character `$`
